geode-core/src/main/java/org/apache/geode/internal/process/ControlFileWatchdog.java

```diff
@@ -146,6 +147,12 @@ private void stopMe() {
     }
   }
 
+  private void deleteExistingStatusResponseFile() throws IllegalStateException {
+    if (!file.delete() && file.exists()) {
```

Comment:
I'd suggest using `File.delete(Path)` which will give you the actual exception in case of failure to delete.

Reply:
Modified, although the call to this method is in a separate thread now, so it logs the exception as a warning.  Maybe it should be info level.